sql/core/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -401,6 +401,14 @@ class Column(protected[sql] val expr: Expression) extends Logging {
   }
 
   /**
+   * True if the current expression is NaN or null
+   *
+   * @group expr_ops
+   * @since 1.5.0
+   */
+  def isNaN: Column = IsNaN(expr)
```

Comment:
can you add one to functions.scala as well? Also add the Python API if you can. Thanks.